container-engine-lib/lib/backend_impls/metrics_reporting/metrics_reporting_kurtosis_backend.go

```diff
@@ -378,81 +377,6 @@ func (backend *MetricsReportingKurtosisBackend) DestroyUserServices(
 	return successes, failures, nil
 }
 
-func (backend *MetricsReportingKurtosisBackend) CreateNetworkingSidecar(
```

Comment:
The networking sidecar was used only for running the traffic control command to modify the internal network and generate the partitions, we remove the sidecar because there isn't any other use case.